### PR TITLE
add schema on edge icon

### PR DESCRIPTION
**Pull request by elfenheart**
screen shot 2015-08-26 at 5 43 49 pm

### CODE REVIEW

File: cdap-ui/app/styles/fonts/icomoon.svg [hunk 1]
```diff
@@ -35,4 +35,5 @@
...
+<glyph unicode="&#xe61c;" glyph-name="SchemaEdge" d="M100.96 755.68v-718.688h718.72v-100.928h-718.72c-27.264 0-50.88 9.984-70.912 30.016s-30.048 43.648-30.048 70.88v718.72h100.96zM993.952 929.92c-20.032 20.032-43.648 30.048-70.912 30.048h-583.328c-27.264 0-51.296-10.016-72.096-30.048s-31.232-43.68-31.232-70.912v-615.328c0-27.296 10.432-51.296 31.232-72.096 20.832-20.832 44.864-31.264 72.096-31.264h583.328c27.264 0 50.88 10.4 70.912 31.264 20.032 20.8 30.048 44.8 30.048 72.096v615.328c0 27.232-10.016 50.88-30.048 70.912zM565.248 460.256v68.96h137.984v-68.96h-137.984zM703.264 392v-68.96h-137.984v68.96h137.984zM565.248 600v68.96h137.984v-68.96h-137.984zM565.248 737.216v68.96h137.984v-68.96h-137.984zM358.176 806.176h138.112v-68.96h-138.112v68.96zM358.176 668.96h138.112v-68.96h-138.112v68.96zM358.176 529.216h138.112v-68.96h-138.112v68.96zM358.176 392h138.112v-68.96h-138.112v68.96zM902.144 323.040h-129.92v68.96h129.92v-68.96zM902.144 460.256h-129.92v68.96h129.92v-68.96zM902.144 600h-129.92v68.96h129.92v-68.96zM902.144 737.216h-129.92v68.96h129.92v-68.96z" />
 </font></defs></svg>
```
Comment (anwar6953) [inline cdap-ui/app/styles/fonts/icomoon.svg]: add newline to end of file

Comment (ajainarayanan) [inline cdap-ui/app/styles/fonts/icomoon.svg]: @anwar6953 This is a generated file that will most likely not be opened by anyone. Not sure if it is required here

File: cdap-ui/app/directives/plumb/my-plumb.less [hunk 2]
```diff
@@ -112,10 +112,20 @@ my-plumb {
     }
   }
 
-  .label-container {
-    background-color: white;
-    padding: 5px;
-    border-radius: 50%;
+  div._jsPlumb_overlay {
+    div.label-container {
+      background-color: @cdap-orange;
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/plumb/my-plumb.less]: Can we have ```cursor: pointer``` so that it appears as if something that the user can click?

**Comment by ajainarayanan** [comment]
:+1: minor comments but otherwise LGTM (apart from @anwar6953's comments)

File: cdap-ui/app/directives/plumb/my-plumb.less (hunk 2 above)
Comment (elfenheart) [inline cdap-ui/app/directives/plumb/my-plumb.less]: added

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**